src/mlnet/mlnet.csproj

```diff
@@ -106,4 +103,13 @@
     </None>
   </ItemGroup>
 
+  <!-- place extra package assets that we need for CLI -->
+  <Target Name="PrepareMlNetCLIPackageAssets" BeforeTargets="PreparePackageAssets">
+    <ItemGroup>
+      <PackageAsset Include="$(TargetPath)" RelativePath="mlnet\tools\netcoreapp2.1\any\" />
+      <PackageAsset Include="$(TargetDir)$(AssemblyName).deps.json" RelativePath="mlnet\tools\netcoreapp2.1\any\" />
+      <PackageAsset Include="$(TargetDir)$(AssemblyName).runtimeconfig.json" RelativePath="mlnet\tools\netcoreapp2.1\any\" />
+      <PackageAsset Include="$(TargetDir)\nlog.config" RelativePath="mlnet\tools\netcoreapp2.1\any\" />
```

Comment:
You can get rid of the duplicated `mlnet\tools\netcoreapp2.1\any\` by making a property:

```xml
  <!-- place extra package assets that we need for CLI -->
  <Target Name="PrepareMlNetCLIPackageAssets" BeforeTargets="PreparePackageAssets">
    <PropertyGroup>
      <MlNetPackagePath>mlnet\tools\netcoreapp2.1\any\</MlNetPackagePath>
    </PropertyGroup>
    <ItemGroup>
      <PackageAsset Include="$(TargetPath)" RelativePath="$(MlNetPackagePath)" />
      <PackageAsset Include="$(ProjectDepsFilePath)" RelativePath="$(MlNetPackagePath)" />
      <PackageAsset Include="$(ProjectRuntimeConfigFilePath)" RelativePath="$(MlNetPackagePath)" />
      <PackageAsset Include="$(TargetDir)\nlog.config" RelativePath="$(MlNetPackagePath)" />
    </ItemGroup>
  </Target>
```